src/adapters/swagger20.es6

```diff
@@ -0,0 +1,149 @@
+import _ from 'underscore';
+
+import { registry, MemberType } from 'minim';
+import '../refract/api';
+
+// Define API Description elements
+const Category = registry.getElementClass('category');
+const Resource = registry.getElementClass('resource');
+const Transition = registry.getElementClass('transition');
+const HttpTransaction = registry.getElementClass('httpTransaction');
+const HttpRequest = registry.getElementClass('httpRequest');
+const HttpResponse = registry.getElementClass('httpResponse');
+const HrefVariables = registry.getElementClass('hrefVariables');
+const Asset = registry.getElementClass('asset');
+
+export const name = 'swagger20';
+
+// TODO: Figure out media type for Swagger 2.0
+export const mediaTypes = [];
```

Comment:
There's some info here, but they don't seem to have the final variants figured out: https://github.com/swagger-api/swagger-spec/issues/110. Apparently the White House is using `application/swagger+json` already today.